docs/src/OptionsPage.js

```diff
@@ -693,6 +693,12 @@ class OptionsPage extends React.Component {
           <td><code>{`false`}</code></td>
           <td>If true, line from cluster to time axis is displayed when content overflows.</td>
         </tr>
+        <tr parent="cluster">
+          <td className="indent">cluster.showStipes</td>
```

Comment:
1) `showStipes` -> `fitOnDoubleClick`
2) Probably, this variable needs to be rephrased (for ex., `deisableFitOnDoubleClick`) and all the logic around it should be neglected.

``` javascript
if (!this.options.deisableFitOnDoubleClick) {
  this.dom.box.ondblclick = ClusterItem.prototype._onDoubleClick.bind(this);
}
```

It prevents us from introducing any breaking changes.

Reply:
I think it would be enough to make 'fitOnDoubleClick' true by default.